samples/gaussian-blur/gaussian-blur.cpp

```diff
@@ -147,7 +147,7 @@ int main(int argc, char* argv[]) {
                                     itemID.get_global(1) + y);
             newPixel +=
                 inPtr.read(inputCoords, smpl) *
-                exp(-1.f * (x * x + y * y) / (2 * stddev * stddev)) /
+                std::exp(-1.f * (x * x + y * y) / (2 * stddev * stddev)) /
```

Comment:
Could you change this to cl::sycl::exp? I have a funny feeling this might not work everywhere (for example, Windows, though I've not tested). Technically, there's no guarantee that std:: code will work inside kernels, though a large amount certainly can, so for this sample I'd prefer that it use the standard SYCL operation.

Reply:
Sure, updated. Rebuilt to be sure, should be working.